### PR TITLE
Task-42849: Add css for share activity icon and shared activity templates

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
This PR will add css for share activity icon and shared activity templates

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 1]
```diff
@@ -1640,6 +1640,13 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       border-top: none!important;
     }
 
+    .shareActivityContainer {
+      margin-left:16px;
+      margin-right: 23px;
+      margin-bottom: 10px;
+      border: 1px solid #e1e8ee;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less]: use a color variable instead

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 2]
```diff
@@ -1640,6 +1640,13 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       border-top: none!important;
     }
 
+    .shareActivityContainer {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less:1643]: Try to use css util classes

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less (hunk 2 above)
Comment (hassenelkamel-exo) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less:1643]: it is in gtmpl, we cannot use it `UISharedDefaultActivity.gtmpl`

**Review by azayati — DISMISSED** [review]
LGTM until QA validation

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**